### DISCUSSION TITLE
Book Launch: Single Source experiment

### DISCUSSION THREAD

**Discussion by mrchristian** [singlesourcepub/community#31] [Announcements]
Hi,

A book launch for 'Citizen Science Skilling for Library Staff, Researchers, and the Public' https://cs4rl.github.io/guide/

From LIBER Citizen Science Working Group

At TIB we are putting together a Single Source Pipeline (alpha) 'ADA Pipeline' where we output the book at Pages Web, Web, EPUB, PDF, PoD, and a variety of sources. The pipeline is currently a manual assemblage of platforms - but put in place so we can see how to automate things.

The key platforms are Fidus Writer, Vivliostyle, and GitHub. But we'll be adding GitLab as a publishing repository, and Thoth as SSP metadata handler.

The publication is OA but we distribute PoD with Ingram Lightningsource and Amazon, and eBooks with Ingram Spark to various distro sites, Google Play, Kindle, etc.

**Comment by xldrkp** [comment]
Sounds great!
